src/ispyb/cli/job.py

```diff
@@ -0,0 +1,471 @@
+#
+# ispyb.job
+#   Get/set information about processing jobs
+#
+
+import re
+import os
+import sys
+from optparse import SUPPRESS_HELP, OptionGroup, OptionParser
+
+import ispyb
+
+try:
+    import procrunner
+    import zocalo
+except ModuleNotFoundError:
+    zocalo = None
+
+# Create a new processing job:
+#   ispyb.job --new --display "Dataprocessor 2000" --comment "The best program in the universe" \
+#             --recipe dp2000 --add-param "spacegroup:P 21 21 21" --add-sweep 1234:1:600
+
+# Display stored information:
+#   ispyb.job 73
+#   ispyb.job 73 -v  # show full record
+
+# Create new processing program row:
+#   ispyb.job 73 -c -p "program" -s "starting up..."
+
+# Update stored information:
+#   ispyb.job 73 -u 1234 -s "running..."
+#   ispyb.job 73 -u 1234 -s "things are happening" --update-time "2017-08-25"
+#   ispyb.job 73 -u 1234 -s "completed successfully" -r success
+#   ispyb.job 73 -u 1234 -s "everything is broken" -r failure
```

Comment:
This should probably be a module-level docstring and/or command line help string.

Reply:
done